### PR TITLE
Tab buttons, bugfixes and more

### PR DESCRIPTION
**Pull request by MigDinny**
A "}" was missing in style.css and I wrote it back there.
I created a new CSS class for tab buttons (close etc). Those buttons have now a margin-left for a better view.
Those buttons were changed to font-awesome 4.7.0. The task says that I should make this responsive between styles but I am unable to change the button class depending on the selected user theme (--> any suggestion is welcome).

I tried to add a tooltip made in HTML + CSS but it was hard to incorporate it in the code because it's hard to find a nice position for the tooltip to appear responsively to the screen and mouse coordinates. The best method I found was to use html tag property "title" which wasn't probably what the task proposed but I think it's pretty suitable anyway. If you guys really want me to do a tooltip in CSS, I'll need some guidance on how to do it (not the design itself because that's not really my problem) because I don't know how I can position the tooltip div and fit perfectly like html "title" does.

Added a dependency for the font-awesome on headers.inc.php and added its folder on assets but didn't remove the older version of font-awesome because it can be linked to something on the project and removing it would probably cause unexpected problems.

Any suggestion is welcome!
GCI task: https://codein.withgoogle.com/dashboard/task-instances/6192065211793408/

### CODE REVIEW

**Comment by robbyoconnor** [comment]
I like use of the title tag attribute :-)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+sites/default/sqlconf.php
```
Comment (robbyoconnor) [inline .gitignore]: Can't do this.

Comment (MigDinny) [reply .gitignore]: It's good now, i guess.

**Comment by MigDinny** [comment]
Done :)

**Comment by teryhill** [comment]
@migdinny can you supply an actual screen shot of what your changes look like?

**Comment by MigDinny** [comment]
image

My mouse is hovering over the refresh button on the right tab. @teryhill

**Comment by teryhill** [comment]
@migdinny Could you do this using the existing version of font-awesome?

**Comment by MigDinny** [comment]
I can, but some icons may not be available becaue the new version brings 41 new icons. The current icons which are in use won't be affected. Do you want me to commit a change removing the new font-awesome version and changing the dependency to the old version?

**Comment by teryhill** [comment]
If you could it would be Great. We will look at upgrading that asset after GCI is over. You are welcome to keep enhancing the code after GCI. We can always use more developers.

**Comment by teryhill** [comment]
Let him get it working with the current assets. We do not need to add assets , if we can help it till we have analyzed the system wide impact and we don't need 40 copies of them. Look at the jquery version that are currently out there.

**Comment by robbyoconnor** [comment]
Alrighty

**Comment by MigDinny** [comment]
@robbyoconnor @teryhill a new commit was added. check it out :)

**Comment by robbyoconnor** [comment]
Could you show screenshots of what it looks like now?

**Comment by MigDinny** [comment]
@robbyoconnor Exactly the same as the other screenshot. The icons I used are present on both versions and are exactly the same. No visual change was made on the last commit.

**Comment by teryhill** [comment]
@migdinny is there an icon that goes with this? Why did you put the font-awsome in the header file directly?

**Comment by MigDinny** [comment]
@teryhill didn't understand the first question... I put the dependency in the header file because there are other dependencies there and i thought it was the right place to put it. Do I have to put it somewhere else? If so, where? :)

**Comment by teryhill** [comment]
Forget the first question I answered it my self.

Look for the program that calls tabs_template.php. In that program you will see a call to the headers program that defines some elements of an array. You will need to add your asset so that it can be called in the same manner as the other ones listed.

**Comment by MigDinny** [comment]
@teryhill is it good now? check the last commit

**Comment by teryhill** [comment]
Very Good @migdinny.

**Comment by MigDinny** [comment]
@teryhill thanks :) but the task on GCI is still unreviewed :c

File: library/headers.inc.php [hunk 2]
```diff
@@ -95,4 +99,4 @@ function resolveFancyboxCompatibility(){ ?>
 //The following seem to always get called.  I don't believe this is proper.
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/dialog.js"></script>
```
Comment (aethelwulffe) [inline library/headers.inc.php]: Add newline please.

**Comment by tmccormi** [comment]
Are we only implementing change is the style_tan.css?  what about the other style sheets?

**Comment by teryhill** [comment]
@tmccormi light and purple are there

File: library/headers.inc.php (hunk 2 above)
Comment (MigDinny) [reply library/headers.inc.php]: Added.

**Comment by MigDinny** [comment]
@aethelwulffe already solved the change you've asked for but there is a conflict and i don't know how to resolve it

**Comment by MigDinny** [comment]
@aethelwulffe can you review?

**Comment by aethelwulffe** [comment]
I already checked this stuff a couple of weeks ago.  I haven't been feeling too good for a while now.  So sorry I didn't come back to this.  I will test one final time now, but we have conflicts that must be resolved before merge.

```
<<<<<<< tab-buttons
    if (in_array('font-awesome', $library_array)) {?>
        <link rel="stylesheet" href="<?php echo $GLOBALS['assets'] ?>/fonts/font-awesome-4-6-3/css/font-awesome.min.css" type="text/css">
=======
    if (in_array("jquery-ui",$library_array)){   ?>
        <link rel="stylesheet" href="<?php echo $GLOBALS['css_path']; ?>jquery-ui-1-12-1/jquery-ui.css" media="screen" />
        <script type="text/javascript" src="<?php echo $GLOBALS['standard_js_path']; ?>jquery-ui-1-12-1/jquery-ui.js"></script>
>>>>>>> master

Just edit out the tab-buttons and master markers, then add in the closing "}" that would be missing.  That should get it.
```

**Comment by MigDinny** [comment]
@aethelwulffe done!

**Comment by aethelwulffe** [comment]
I hope this doesn't cause other conflicts, but that will just be more good training! :)
Merging up.

**Pull request merged**